simplq/src/components/common/QrScanner/QrScanner.js

```diff
@@ -0,0 +1,47 @@
+// Component responsible for the /scanQr route
+
+import React, { useEffect } from 'react';
+import QrReader from 'react-qr-scanner';
+import styles from './QrScanner.module.scss';
+
+export default () => {
+  useEffect(() => {
+    window.scrollTo('40px', '0px');
+  });
+
+  const handleScan = (data) => {
+    if (data != null) {
+      const res = this.getRoute(window.location.origin, data.text);
```

Comment:
As pointed out here, 'this' seems to be a bug we missed during refactoring. Could you note this in an issue?

Reply:
Yo, raising here: https://github.com/SimplQ/simplQ-frontend/issues/660